A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_CRZ.js

```diff
@@ -0,0 +1,122 @@
+            let leftConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:1", "gallon") * unitFactor * 0.001;
+            let rightConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:2", "gallon") * unitFactor * 0.001;
+
+            this.FuelUsedTotal.textContent = fastToFixed(leftConsumption+rightConsumption, 0);
+            this.FuelUsedLeft.textContent = fastToFixed(leftConsumption, 0);
+            this.FuelUsedRight.textContent = fastToFixed(rightConsumption, 0);
+
+            // Oil
+            var value = SimVar.GetSimVarValue("ENG OIL QUANTITY:1", "percent") * 0.01 * 25;
+            this.OilQuantityLeft.textContent = Math.floor(value) + ".";
+            this.OilQuantityLeftDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            value = SimVar.GetSimVarValue("ENG OIL QUANTITY:2", "percent") * 0.01 * 25;
+            this.OilQuantityRight.textContent = Math.floor(value) + ".";
+            this.OilQuantityRightDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            // Engines
+            value = SimVar.GetSimVarValue("TURB ENG VIBRATION:1", "Number");
+            if (value < 0) value = 0.0;
+            this.VibN1Left.textContent = Math.floor(value) + ".";
+            this.VibN1LeftDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            value = SimVar.GetSimVarValue("TURB ENG VIBRATION:2", "Number");
+            if (value < 0) value = 0.0;
+            this.VibN1Right.textContent = Math.floor(value) + ".";
+            this.VibN1RightDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            value = SimVar.GetSimVarValue("TURB ENG VIBRATION:1", "Number");
+            if (value < 0) value = 0.0;
+            this.VibN2Left.textContent = Math.floor(value) + ".";
+            this.VibN2LeftDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            value = SimVar.GetSimVarValue("TURB ENG VIBRATION:2", "Number");
+            if (value < 0) value = 0.0;
+            this.VibN2Right.textContent = Math.floor(value) + ".";
+            this.VibN2RightDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+
+            // Cabin pressure
+            value = SimVar.GetSimVarValue("PRESSURIZATION CABIN ALTITUDE RATE", "feet per second");
+            this.CabinVerticalSpeed.textContent = fastToFixed(value, 0);
+            
+            value = SimVar.GetSimVarValue("PRESSURIZATION CABIN ALTITUDE", "feet");
+            this.CabinAltitude.textContent = fastToFixed(value, 0);
+
+            var cabinAltMetres = value * 0.3048;
+            var cabinPressurePascal = 101325 * Math.exp(-0.00011857591*cabinAltMetres);
+            var cabinPressurePSI = cabinPressurePascal * 0.000145038;
+            var outsidePressureINHG = SimVar.GetSimVarValue("AMBIENT PRESSURE", "inHg");
+            var outsidePressurePSI = outsidePressureINHG * 0.491154;
+            var pressureDiff = cabinPressurePSI - outsidePressurePSI;
```

Comment:
Use `const` as those do not change, and fix spacing in `Math.exp(...)` at line 107